test/Dapr.Extensions.Configuration.Test/DaprSecretStoreConfigurationProviderTest.cs

```diff
@@ -437,7 +437,7 @@ private async Task SendBulkResponseWithSecrets(Dictionary<string, string> secret
             {
                 var secretsResponse = new Autogenerated.SecretResponse();
                 secretsResponse.Secrets[secret.Key] = secret.Value;
-                getBulkSecretResponse.Data.Add(secret.Key, secretsResponse);
+                getBulkSecretResponse.Data.Add("IgnoredKey" + secret.Key, secretsResponse);
```

Comment:
Why this change in particular?

I don't quite get the connection between this and your fix. Can you add a comment?

Reply:
Bulk Secret API returns `Dictionary<string,Dictionary<string,string>>`. The outer Dictonary values are ignored by `DaprSecretStoreConfigurationProvider`. In the test setup the inner and outer values are identically. This is not the case in real world usage. Therefore the test did not fail to highlight an issue. This change is to ensure the tests are failing when reading the outer dictonary key, as I did.

Reply:
Thanks, can you please add a comment?

Reply:
Done